dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -252,7 +252,8 @@
     /* Validation Results API */
     E7500( "Organisation unit does not exist: `{0}`" ),
     E7501( "Validation rule does not exist: `{0}`" ),
-    E7502( "Query filter for period is not a valid ISO expression: `{0}`" );
+    E7502( "Filter for period is not a valid ISO expression: `{0}`" ),
```

Comment:
Maybe it is enough to say `Filter for period is not valid`. People could look the period format up in the docs.

Reply:
Changed both messages that were related to ISO expressions.